src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Header/Content/action.html.twig

```diff
@@ -0,0 +1,3 @@
+<div class="float-right grid-actions-header-text">
+  <span>{{ column.name }}</span>
```

Comment:
🤔 What is the use of this twig template ?

Reply:
it overrides grids actions header text- without this template, it would be just a plain text with no unique class to identify it. 

With that, instead of just always floating the last element in the grid I'm targeting the specific case I need : **To float the text only in actions column**. And I added some more commits which now floats only then, when the actions column is in the last place of the grid :smiley:

Reply:
I'm wondering ... we currently have a `default.html.twig` header with a `if` inside:
https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Header/Content/default.html.twig#L30
and another usecase where we use the name of the template to override:
https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Header/Content/position_handle.html.twig

Should we keep using name overriding or add more `if` in the default file ?

Reply:
well , adding an if statement just to add class in twig seems for me not so clean approach - I would rather prefer the template override as it is now, since if someone would like to change the class name or something in that template then that user won't have to look throw if statement to find out whats going on there :thinking:

Reply:
Fair point 👍